package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fulcrum-query-sql",
-  "version": "0.5.6",
+  "version": "0.5.7",
```

Comment:
Let's call this 0.6.0 since this adds new functionality. It's not really a bug fix.